Modules/Sources/WordPressReader/Resources/gutenbergCommentHeadTemplate.html

```diff
@@ -4,25 +4,4 @@
     <style>
         %2$@
     </style>
-    <script>
-        function debounce(fn, timeout) {
```

Comment:
I disabled content JavaScript for comments for safety and performance. We will not be tracking these events astray are daily useless, so I removed it.